python/python/glide/routes.py

```diff
@@ -48,6 +48,18 @@ def __init__(self, slot_type: SlotType, slot_id: int) -> None:
         self.slot_id = slot_id
 
 
+class ByAddressRoute(Route):
+    def __init__(self, host: str, port: Optional[int] = None) -> None:
```

Comment:
we need to specify the format of the host (ip:tcp_port) and handle erroneous values, or make host -> host_ip.
Document that if port is None than a default will be used